contracts/src/main/kotlin/com/r3/corda/lib/tokens/contracts/states/FungibleToken.kt

```diff
@@ -31,9 +33,10 @@ import net.corda.core.schemas.QueryableState
  * @param T the [TokenType] this [FungibleToken] state is in respect of.
  */
 @BelongsToContract(FungibleTokenContract::class)
-open class FungibleToken<T : TokenType>(
+class FungibleToken<T : TokenType>(
```

Comment:
NonFungibleToken is still open with one final field is it on purpose?

Reply:
We can open it up later if necessary.